install/kubernetes/helm/istio/values.yaml

```diff
@@ -439,8 +439,8 @@ global:
   # The second network, `network2`, in this example is defined differently with all endpoints
   # retrieved through the specified Multi-Cluster registry being mapped to network2. The
   # gateway is also defined differently with the name of the gateway service on the remote
-  # cluster. The public IP for the gateway will be determined from that remote service (not
-  # supported yet).
+  # cluster. The public IP for the gateway will be determined from that remote service (only
+  # LoadBalancer gateway service type is currently supported).
```

Comment:
I guess you mean only LoadBalancer is currently supported for automatic IP configuration. For a NodePort gateway service, it still can be configured manually.

Reply:
Yes, for a NodePort type service, we can not get the ips in program

Reply:
updated